### ISSUE TITLE
server side

### ISSUE THREAD

**Issue by Cyclopropinon** [bishan-batel/TintedBeacon#1]
hi, i wanna add this to my own server, is it possoble to make it server-side only?

**Comment by bishan-batel** [comment]
I don't believe so with the method that I'm currently using, as the check for the rendering the beacon beam happens entirely on the client side. There is a server side check for apply the particle effect, but there is no communication between the server & the client about whether or not to render. The only method I could think of to remedy this is the server sending the client a ghost block where the beacon is. I'll see if I could try that in an update, sorry for reading this issue so late - this is the first one I've gotten on my own repository.
